src/components/index/Header.js

```diff
@@ -100,12 +100,12 @@ const Header = () => (
                         <img className="banner__image" src={TheiaConLogo}></img>
                     </a>
                     <h1 style={{ fontSize: '2.2rem' }}>
-                        The Call for Presentations is now OPEN for the <a href="https://events.eclipse.org/2023/theiacon/">2023 TheiaCon Event</a>!
-                        Submit your proposal today to be a speaker at the largest Theia event of the year!
+                        <a href="https://events.eclipse.org/2023/theiacon/">TheiaCon 2023</a> is approaching! | VIRTUAL EVENT | November 15 - 16, 2023
```

Comment:
minor: do we want a `|` to separate the date, else it looks out of place?

```suggestion
                        <a href="https://events.eclipse.org/2023/theiacon/">TheiaCon 2023</a> is approaching! | VIRTUAL EVENT | November 15 - 16, 2023 | 
```

![image](https://github.com/eclipse-theia/theia-website/assets/40359487/779dfbda-09ff-4e8b-bbaa-f528ba078616)